### PR TITLE
Addition of PFSUM aggregator, will replace missing values with previous value in series

### PR DESCRIPTION
**Pull request by SumanjeetBhatti**
Merge changes for #851 new requirement changes

### CODE REVIEW

**Comment by johann8384** [comment]
It looks like you might have a mix of tabs/spaces there, can you check your editor and make sure the settings are to indent with 2 space, no tabs. Thanks!

http://opentsdb.net/docs/build/html/development/index.html

File: src/core/AggregationIterator.java [hunk 1]
```diff
@@ -653,6 +653,9 @@ public long nextLongValue() {
         case MIN:
           r = Long.MIN_VALUE;
           break;
+		case PREV:
```
Comment (johann8384) [inline src/core/AggregationIterator.java]: Tabs/Spaces Here, make sure your IDE is set to "2 space indent/no tabs"

File: src/core/Aggregators.java [hunk 2]
```diff
@@ -36,12 +36,20 @@
     LERP,   /* Regular linear interpolation */
     ZIM,    /* Returns 0 when a data point is missing */
     MAX,    /* Returns the <type>.MaxValue when a data point is missing */
-    MIN     /* Returns the <type>.MinValue when a data point is missing */
+    MIN,     /* Returns the <type>.MinValue when a data point is missing */
+	PREV    /* Returns the previous value stored, when a data point is missing */
```
Comment (johann8384) [inline src/core/Aggregators.java]: Same here

**Comment by manolama** [comment]
Merged in 53cb31e1f5bac277f1e4fd79fddea4b77451ab2d. Thanks!

**Pull request closed without merging**